server.go

```diff
@@ -252,3 +257,122 @@ func successfulLogin(conf config.Config, delphisBackend backend.DelphisBackend)
 	}
 	return http.HandlerFunc(fn)
 }
+
+func getImage(delphisBackend backend.DelphisBackend) http.Handler {
+	fn := func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != "POST" {
+			logrus.WithError(errors.New("non-POST request was sent to getImage"))
+			w.WriteHeader(http.StatusBadRequest)
+		}
+
+		body := map[string]string{}
+		if err := json.NewDecoder(r.Body).Decode(&body); err != nil {
+			logrus.WithError(err).Error("failed to decode request body for getImage")
+			w.WriteHeader(http.StatusInternalServerError)
+			if _, err := w.Write([]byte(fmt.Sprintf("500 - Failed to decode request body!"))); err != nil {
+				return
+			}
+			return
+		}
+		logrus.Debugf("Body: %v\n", body)
+		if _, ok := body["media_id"]; !ok {
+			w.WriteHeader(http.StatusBadRequest)
+		}
+
+		if _, ok := body["media_type"]; !ok {
+			w.WriteHeader(http.StatusBadRequest)
+		}
+
+		mediaBytes, err := delphisBackend.GetMedia(r.Context(), body["media_id"], body["media_type"])
```

Comment:
I think these can just be in the path for the get like `<domain>/images/<uuid>.<extension>` or something.